src/hotspot/share/utilities/numberSeq.cpp

```diff
@@ -110,6 +110,29 @@ double AbsSeq::dsd() const {
   return sqrt(var);
 }
 
+
+void AbsSeq::merge(AbsSeq& abs2, bool clear_this) {
+
+  if (num() == 0) return;  // nothing to do
+
+  abs2._num += _num;
+  abs2._sum += _sum;
+  abs2._sum_of_squares += _sum_of_squares;
+
+  // Decaying stats need a bit more thought
+  assert(abs2._alpha == _alpha, "Caution: merge incompatible?");
+  // guarantee(false, "NYI");
```

Comment:
This will expand into setting some breadcrumbs in `abs2`, such that any attempt to query a decayed stat from the object will result in an error.